spec/models/work_packages/scopes/relatable_spec.rb

```diff
@@ -150,6 +151,16 @@
             .to be_empty
         end
       end
+
+      context "with the existing relation and the queried being '#{current_type}' typed but explicitly ignoring the existing" do
+        let(:relation_type) { current_type }
+        let(:ignored_relation) { directly_related_work_package.relations.first }
+
+        it 'is empty' do
```

Comment:
There is a typo here, the spec should be called something like :

```suggestion
        it 'contains the directly related work package' do
```